content/en/blog/2021-04-27-cant-read-wont-buy.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Can't Read, Won't Buy"
+slug: "cant-read-wont-buy"
+date: "2021-04-27T11:24:40-04:00"
+draft: false
+image_webp: images/blog/streetblur.webp
+image: images/blog/streetblur.jpg
+author: Rebecca Bilbro
+description : "In this post we'll explore some of the drawbacks of using only English as a sales medium, check out some creative solutions, identify some areas of caution, and consider what this all means to the future of global app development."
+---
+
+To say we are not at our most empowered when trying to communicate in an unfamiliar language is a bit of an understatement. As if the limitations of one's own language weren't bad enough, trying to get along in a different language often makes you feel downright *helpless*. Research suggests these feelings also strongly inform consumers' buying decisions. What does this mean for companies who want to connect with users overseas? In this post we'll explore some of the drawbacks of using English-only as a sales medium, check out some creative solutions, and consider what this all means for global app development.
+
+## Why Not Just English?
+
+In David Sedaris' (2000) ["Me Talk Pretty One Day"](https://en.wikipedia.org/wiki/Me_Talk_Pretty_One_Day), the author recounts his time as a new student in a French language class for adults:
+
+> The first day of class was nerve-racking because I knew I'd be expected to perform. [The teacher] proceeded to rattle off a series of administrative announcements: "If you have not *meimslsxp* or *lgpdmurct* by this time, then you should NOT be in this room. Has everyone *apzkiubjxow*? Everyone? Good, we shall begin."
+
+The administrative context is enough to transform what would otherwise be a low-stakes situation into a stressful one &mdash; it's one thing to miss every few words of a menu or a movie in a different language, and quite another to not fully understand a legal, political, or financial choice due to unfluency.
+
+In [survey of international consumers and firms](https://insights.csa-research.com/reportaction/305013126/Marketing), Donald DePalma, Benjamin Sargent, and Renato Beninatto found that that language has a big impact on global consumer buying decisions;
```

Comment:
just saw an extra "that" in the above sentence ☹️

Reply:
Sorry for missing it :-(

Reply:
There are markdown editors that support writing such as [typora](https://typora.io/) -- but I don't see a lot on their site about spell and grammar check. People are [asking for Grammarly on Typora](https://github.com/typora/typora-issues/issues/1453) -- which might be an interesting solution. This medium article [talks about creating a Grammarly extension for VSCode](https://medium.com/@znck/grammarly-in-code-b62dd1a4c76e) - it would be worth checking out; I like Grammarly in the browser, but am worried that a homegrown extension might be slow. 

I'll keep looking!